login-fire-common-behavior.html

```diff
@@ -264,6 +264,10 @@
         observer: '_userChanged'
       },
 
+      /**
+       *    This property is used in login-fire and login-fire-social when the user is signed in
+       *    and is allowed to sign out. It then shows just one button instead of multiple to sign out.
+       */
       _showOneButton: {
```

Comment:
I agree I think the issue I dealt with was there not being a shared behavior just between login-fire and login-fire-social and not the others. The better way would have been to create another behavior for that since login-fire-button doesn't use it.

We can call it login-fire-aggregator-behavior or something. I created #89 to follow this.